tools/cbmc/proofs/HTTP/IotHttpsClient_ReadContentLength/IotHttpsClient_ReadContentLength_harness.c

```diff
@@ -12,8 +12,11 @@
 void harness() {
   IotHttpsResponseHandle_t respHandle = allocate_IotResponseHandle();
   initialize_IotResponseHandle(respHandle);
-  if (respHandle)
+  if (respHandle) {
     __CPROVER_assume(is_valid_IotResponseHandle(respHandle));
+    // ???: replace assumption above with initialization
+    respHandle->httpParserInfo.readHeaderParser.data = respHandle;
```

Comment:
Would this be better in `allocate_IotResponseHandle()`?

Reply:
Yes, this is the CBMC issue I had to work around: I took the assumption from is_valid_IoTResponseHandle and copied it here an assignment in the test harness.  I could put it in allocate_IotResponseHandle as you suggest, but I'd like to keep it here so we remember to delete it when CBMC is fixed.